### PR TITLE
[475 477] : metaverse switching

### LINKED ISSUE #475: As a user I want to be able to select between available metaverses from the dropdown in My Properties page so I can browse all available properties on the platform
The metaverse dropdown would be the main filter. All other functionalities in the explore page would be dictated by this filter.
We need to enable it to pull all available metaverse collections we support and list them.

AC1: When I click the dropdown, then all available metaverse collections should be listed and I can choose and click those.
AC2: When I click a certain dropdown option, then the properties are filtered according to the collection/metaverse I chose.

### PR DESCRIPTION
**Pull request by mishaadzh**
**Detailed description**:
Added metaverse select, properties count in the tab

**Which issue(s) this PR fixes**:
Fixes #475  #477 #468 #470

**Special notes for your reviewer**:

**Checklist**
- [x] Properties count
- [x] Metaverse dropdown

### CODE REVIEW

**Review by xdemocle — APPROVED** [review]
great job

**Comment by failfmi** [comment]
Looks good, I have a few more suggestions regarding the generalisation.
Currently, in the graph we store all the metaverses that are currently supported:
Instead of hard-coding in the code the metaverses with their IDs (Decentraland, CryptoVoxels, etc.), we can query the graph and fetch them:
```gql
{
  metaverses {
    id
    name
  }
}
```
This way, upon loading the component, we fetch the metaverses and populate them in the dropdown. After that, based on the metaverse selected, we modify the queries to fetch only assets/rents related to that metaverse, rather than fetching everything and just filter it based on the selected one. Eventually, a lot of assets might be listed in LandWorks and fetching everything might lead to loading issues.

**Comment by xdemocle** [comment]
#470 added to description

File: src/modules/land-works/api.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const ASSET_SUBSCRIPTION = gql`
 `;
 
 export const USER_SUBSCRIPTION = gql`
-  subscription GetUser($id: String) {
+  subscription GetUser($id: String, $metaverse: String) {
     user(id: $id) {
       id
       consumerTo {
```
Comment (failfmi) [inline src/modules/land-works/api.ts:77]: `metaverse` should be filtered here as well

Comment (failfmi) [reply src/modules/land-works/api.ts:77]: payment token here as well

Comment (failfmi) [reply src/modules/land-works/api.ts:77]: payment token here as well

Comment (mishaadzh) [reply src/modules/land-works/api.ts:77]: this has already been implemented

Comment (failfmi) [reply src/modules/land-works/api.ts:77]: This has to be added to `consumerTo` similar to `assets`.

**Review by failfmi — APPROVED** [review]

**Pull request merged**